src/main/java/seedu/address/logic/commands/AssignCommand.java

```diff
@@ -44,9 +44,13 @@
             + "[INDEX]...";
 
     public static final String MESSAGE_SUCCESS = "Players successfully assigned to team.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the team";
+    public static final String MESSAGE_FAILURE = "Not all players have been successfully processed.";
+    public static final String MESSAGE_DUPLICATE_PERSON =
+            "\n%1$s: There is already the same player that exists in the team.";
+    public static final String MESSAGE_NO_TEAM_TO_UNASSIGN = "\n%1$s: Cannot unassign player that are not in a team.";
```

Comment:
grammar edit: "Cannot unassign player that is not in a team"

Reply:
Thank you! Will make this change accordingly.